velox/substrait/tests/data/q6_first_stage.json

```diff
@@ -67,12 +67,21 @@
                         "input": {
                             "project": {
                                 "common": {
-                                    "direct": {}
+                                    "emit":{
+                                        "outputMapping":[
+                                           2
+                                        ]
+                                    }
                                 },
                                 "input": {
                                     "project": {
                                         "common": {
-                                            "direct": {}
+                                            "emit":{
+                                                "outputMapping":[
+                                                   4,
+                                                   5
+                                                ]
+                                            }
```

Comment:
It selects the two columns for the projection and they become the 4th and 5th indexes of total 6 column table. In order to retain the test case accuracy which previously assumed that the Substrait projection only output projection columns, we add emit to select output values. This supports the description in the PR. 

cc @mbasmanova